src/einspline/bspline_create_cuda.cu

```diff
@@ -261,13 +262,7 @@ create_UBspline_3d_s_cuda_conv (UBspline_3d_d* spline)
 
   size_t size = Nx*Ny*N*sizeof(float);
 
-  cudaMalloc((void**)&(cuda_spline->coefs), size);
-  cudaError_t err = cudaGetLastError();
-  if (err != cudaSuccess) {
-    fprintf (stderr, "Failed to allocate %ld memory for GPU spline coefficients.  Error %s\n",
-	     size, cudaGetErrorString(err));
-    abort();
-  }
+  cudaCheck(cudaMalloc((void**)&(cuda_spline->coefs), size));
```

Comment:
It seems like we're losing most of the specificity in the error output here.  Since cudaCheck throws this can be put back.